app.rb

```diff
@@ -0,0 +1,89 @@
+require_relative('./book')
+require_relative('./student')
+require_relative('./teacher')
+require_relative('./rentals')
+
+class App
+  attr_reader :books, :people, :rentals
+
+  def intialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def books_list
+    books.each { |book| puts `Title: #{book.title}, Author: #{book.author}` }
+  end
+
+  def people_list
+    people.each { |person| puts `Name: #{person.name} ID: #{person.id} Age: #{person.age}` }
+  end
+
+  # rubocop:disable Metrics/MethodLength
+  def add_person
+    puts 'Do you want to add a student (1) or a teacher (2) '
+    person = gets.chomp.to_i
+    puts 'Enter name'
+    name = gets.chomp
+    puts 'Enter age'
+    age = gets.chomp.to_i
+
+    case person
+    when 1
+      puts 'Enter classroom'
+      classroom = gets.chomp
+      puts 'Has parent permission [Y/N]'
+      permission = gets.chomp
+      @people << if permission == 'Y'
+                   Student.new(classroom, age, name, parent_permission: true)
+                 else
+                   Student.new(classroom, age, name, parent_permission: false)
+                 end
+
+    when 2
+      puts 'Enter specialization:'
+      specialization = gets.capitalize.chomp
+      @people << Teacher.new(specialization: specialization, age: age, name: name)
+    else
+      puts 'Invalid choice, please try again'
+    end
+    puts 'Person created successfully'
+  end
+  # rubocop:enable Metrics/MethodLength
+
+  def add_book
+    print 'Enter title:'
+    title = gets.chomp
+    print 'Enter author:'
+    author = gets.chomp
+    books.push(Book.new(title, author))
+    puts 'Book created successfully'
+  end
```

Comment:
-  [ ] When I create a book, your app shows the error below. Please work on that
<img width="611" alt="image" src="https://user-images.githubusercontent.com/77190203/181914613-acd5edff-fe21-454e-9a2c-5e9769e952c4.png">